### ISSUE TITLE
Issue with comma-spacing

### ISSUE THREAD

**Issue by SimeonC** [eslint/eslint#9225]
**Tell us about your environment**

* **ESLint Version:**
4.6.1
* **Node Version:**
8.2.1
* **npm Version:**
5.3.0

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint 7.2.3

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
parser: babel-eslint
extends: airbnb
env:
  browser : true
rules:
  comma-dangle: [2, never]
  max-len:
    - 2
    - 120
    - 2
    - { ignoreUrls: true, ignoreComments: false }
  react/no-find-dom-node: 0
  arrow-body-style: 0
  no-mixed-operators: 0
  no-shadow: [2, { allow: [err, error] }]
  react/prefer-stateless-function: [error, { ignorePureComponents: true }]

  no-console: 2
  no-underscore-dangle: 0
  guard-for-in: 0
  no-param-reassign: [2, { props: false }]

  react/forbid-prop-types: [2, { ignore: ['intl', 'location', 'match', 'history'] }]
  react/prop-types: [2, { ignore: ['intl', 'location', 'match', 'history'] }]

  jsx-a11y/no-static-element-interactions: 0
  jsx-a11y/tabindex-no-positive: 0
  jsx-a11y/img-has-alt: 0
  jsx-a11y/label-has-for: 0

  # Custom rules from ./eslint-rules
  decorator-order: [2, { last: ['inject', 'observer'] }]
```

</details>

**What did you do? Please include the actual source code causing the issue.**

```js


fetch(`https://maps.googleapis.com/maps/api/distancematrix/json?${
  qs.stringify({
    origins: '',
    destination: `place_id:${12312312}`,
    mode: 'walking',
    language: 'en',
    key: config.googleMapsKey
  })
  }`)
.then(response => console.log('distancematrix test', response));
const test = `${1234},${123}`;

```

**What did you expect to happen?**
No Errors to occur.

**What actually happened? Please include the actual, raw output from ESLint.**
```const test = `${1234},${123}`;```
Gives error: `error  A space is required after ','  comma-spacing`

**Comment by not-an-aardvark** [comment]
Thanks for the report. Does this issue also occur with the default parser? I can't seem to reproduce it in the [demo](https://eslint.org/demo/#eyJ0ZXh0IjoiZmV0Y2goYGh0dHBzOi8vbWFwcy5nb29nbGVhcGlzLmNvbS9tYXBzL2FwaS9kaXN0YW5jZW1hdHJpeC9qc29uPyR7XG4gIHFzLnN0cmluZ2lmeSh7XG4gICAgb3JpZ2luczogJycsXG4gICAgZGVzdGluYXRpb246IGBwbGFjZV9pZDokezEyMzEyMzEyfWAsXG4gICAgbW9kZTogJ3dhbGtpbmcnLFxuICAgIGxhbmd1YWdlOiAnZW4nLFxuICAgIGtleTogY29uZmlnLmdvb2dsZU1hcHNLZXlcbiAgfSlcbiAgfWApXG4udGhlbihyZXNwb25zZSA9PiBjb25zb2xlLmxvZygnZGlzdGFuY2VtYXRyaXggdGVzdCcsIHJlc3BvbnNlKSk7XG5jb25zdCB0ZXN0ID0gYCR7MTIzNH0sJHsxMjN9YDsiLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjYsInNvdXJjZVR5cGUiOiJzY3JpcHQiLCJlY21hRmVhdHVyZXMiOnt9fSwicnVsZXMiOnsiY29tbWEtc3BhY2luZyI6Mn0sImVudiI6e319fQ==).

If this only occurs when using `babel-eslint`, it might be an issue with the `babel-eslint` parser.

**Comment by SimeonC** [comment]
@not-an-aardvark Yea, it looks like this is a babel-eslint issue, I'll open a ticket there. I didn't know about the demo, I'll try that next time! Thanks for your help.
